### PR TITLE
OpenStack: Rename techpreview jobs

### PR DESCRIPTION
**Pull request by mandre**
Add a techpreview suffix so that jobs are clearly identifiable.

### CODE REVIEW

**Comment by mandre** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.18-techpreview-e2e-openstack-ovn-techpreview

**Comment by mandre** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.18-techpreview-e2e-openstack-ovn-techpreview

**Comment by mandre** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.18-techpreview-e2e-openstack-ovn-techpreview

**Comment by mandre** [comment]
Depends on https://github.com/openshift/release/pull/57131

File: ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.17__techpreview.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-openstack-ovn-parallel
+- as: e2e-openstack-ovn-techpreview
```
Comment (stephenfin) [inline ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.17__techpreview.yaml:42]: nit: any reason to drop the `-parallel` suffix? Just making sure this isn't e.g. a rebase snafu.

**Review by stephenfin — COMMENTED** [review]
Looks fine. One nit/question inline.

File: ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.17__techpreview.yaml (hunk 1 above)
Comment (mandre) [reply ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.17__techpreview.yaml:42]: Yes, sorry I forgot to mention in the commit message. I removed the `-parallel` suffix to be consistent with the jobs from other platforms.

**Comment by mandre** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.18-techpreview-e2e-openstack-ovn-techpreview

**Pull request merged**